OneCommunityGlobal/HGNRest
# Update env file to store the default password

Hello @EvianTan, As discussed on the slack, can you please verify this. Thank you

# Description
The current implementation of our codebase has a significant security vulnerability, as passwords are hardcoded in multiple locations throughout the code. This means that anyone with access to the repository can easily access these sensitive credentials. To address this issue, we need to implement a more secure approach for storing these passwords. One potential solution is to utilize a separate .env file to store all password information. By doing so, we can ensure that the passwords are kept confidential and are not visible to unauthorized individuals. This approach can significantly reduce the risk of security breaches and protect our system from potential attacks. To implement this solution, we will need to refactor the existing codebase to remove all hardcoded passwords and replace them with calls to the .env file. We will also need to ensure that the .env file is securely stored and that access to it is limited only to authorized individuals. Additionally, we may need to update any associated documentation or processes to reflect this change in our security approach.
Or Implements # (WBS) 

## Related PRS (if any):
No related PR 
…

## Mainly changes explained:
Updated the .env file to inclure the password. (DEF_PWD=123Welcome!)
Made the changes in the places where the password is hardcoded.
…

## How to test:
check into current branch
do `npm install` and `...` to run this PR locally
Check if the login is successgul from different users.
